src/components/layout/DocsLayout.js

```diff
@@ -79,6 +79,12 @@ const Wrapper = styled.div`
 `;
 
 const StyledTableOfContents = styled(TableOfContents)`
+  // space the ToC emoji and text
```

Comment:
Do we not want to do this in `@storybook/design-system`?

Reply:
I don't think so, the API is to pass an arbitrary string to the ToC component. Sometimes that string might not have an emoji, which will mean the first character is positioned in a strange way.

For example, [the Chromatic docs](https://www.chromatic.com/docs/), which will one-day use the ToC component, don't use emoji.